packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/About.5.doc.mdx

```diff
@@ -136,6 +136,33 @@ A column specifies the data to render a cell in the header row of the Flowsheet
 |**columnHighlightColor**|`enum: ["green", "orange"]`|optional|none|The color to be used for highlighting a column.|
 |**columnHighlightDescription**|string|optional|none|The information to be conveyed to screen readers about the highlighted column.|
 
+### Section
+<Notice variant="important" ariaLevel="2">
+Due to poor support in Apple VoiceOver for spanned table headings, Table component sections are not currently recommended for applications used on Apple devices.
+</Notice>
+
+A section defines the rows rendered in the section.
+
+|       Name        |  Type  |                    Required                     | Default Value |                                                     Description                                                     |
+| ----------------- | ------ | ----------------------------------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------- |
+| **id**            | String | <span style={{color:'#d53040'}}>Required</span> | None          | An identifier to uniquely identify the section in the grid.                                                         |
+| **isCollapsible** | Bool   | Optional                                        | False         | A Boolean value indicating whether the the section is collapsible.                                                  |
+| **isCollapsed**   | Bool   | Optional                                        | False         | A Boolean value indicating whether the section is collapsed. If true, the contents of the section is not displayed. |
+| **text**          | String | Optional                                        | None          | A text string to render in the section header.                                                                      |
+| **rows**          | Array  | Optional                                        | []            | An array of row objects to be rendered in the section.                                                              |
+| **subsections**   | Array  | Optional                                        | []            | An array of subsections objects to be rendered within the section.                                                  |
+
+#### Subsection
```

Comment:
Shouldn't it be the same level header like Section: `### Subsection`? Looks smaller than Section and Rows headers.

Reply:
@adoroshk - This is a good call out.  I copied the structure from the Terra Table component where it originates.  However, I could update this style for both components.  I do think that is appropriate.